tests/aws-cpp-sdk-cognitoidentity-integration-tests/CMakeLists.txt

```diff
@@ -38,4 +38,5 @@ target_link_libraries(${PROJECT_NAME} ${PROJECT_LIBS})
 if(MSVC AND BUILD_SHARED_LIBS)
     set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS
             "/DELAYLOAD:aws-cpp-sdk-access-management.dll /DELAYLOAD:aws-cpp-sdk-cognito-identity /DELAYLOAD:aws-cpp-sdk-iam /DELAYLOAD:aws-cpp-sdk-core.dll")
+    target_link_libraries(${PROJECT_NAME} delayimp.lib)
```

Comment:
make build work with non-msvs generators (clion or ninja)
https://discourse.cmake.org/t/delayload-inconsistency-between-visual-studio-and-ninja-generators/2719